src/components/actions/examples/ActionBarExamples.tsx

```diff
@@ -36,6 +36,10 @@ export class ActionBarExamples extends React.Component<any, any> {
           <label className='form-control-label'>Action bar without actions</label>
           <ActionBar actions={actions} />
         </div>
+        <div className='form-group'>
+          <label className='form-control-label'>Action bar with default classes turned off and extra classes</label>
+          <ActionBar actions={actions} removeDefaultContainerClasses={true} extraContainerClasses={['coveo-table-actions-container', 'p2']}/>
```

Comment:
You don't have to set `removeDefaultContainerClasses` to true. You can just have the prop on or off if you want  it or not